packages/jupyter/jupyter.2.2.1/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "1.2"
+name: "jupyter"
+maintainer: [
+  "Akinori ABE <aabe.65535@gmail.com>"
+]
+authors: [
+  "Akinori ABE"
+]
+license: "MIT"
+homepage: "https://akabe.github.io/ocaml-jupyter/"
+bug-reports: "https://github.com/akabe/ocaml-jupyter/issues"
+dev-repo: "https://github.com/akabe/ocaml-jupyter.git"
+
+available: [ ocaml-version >= "4.02.0" ]
+build: [
+  [ "jbuilder" "subst" ] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+depends: [
+  "base-threads"
+  "base-unix"
+  "uuidm" {>= "0.9.6"}
+  "base64" {>= "2.1.2"}
+  "lwt" {>= "3.0.0"}
+  "stdint" {>= "0.4.2"}
+  "zmq" {>= "4.0-8"}
+  "lwt-zmq" {>= "2.1.0"}
+  "yojson" {>= "1.3.3"}
+  "ppx_deriving_yojson" {>= "3.0"}
+  "cryptokit" {>= "1.12"}
+  "ocamlfind" {build & >= "1.5.0"}
```

Comment:
I notice the package uses `jbuilder`. Is `ocamlfind` still a dependency? It might be safe to remove now.